api/Client.json

```diff
@@ -48,6 +48,52 @@
           "deprecated": false
         }
       },
+      "frameType": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Client/frameType",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "43"
+            },
+            "chrome_android": {
+              "version_added": "43"
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "44",
+              "notes": "Service workers (and Push) have been disabled in the Firefox 45 & 52 Extended Support Releases (ESR)."
+            },
+            "firefox_android": {
+              "version_added": "44"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": null
+            },
+            "opera_android": {
+              "version_added": null
```

Comment:
Both versions of Opera should be 30.

Reply:
If I look up the Client#frameType page on MDN Opera states support has unknown? Not too sure what data source to trust?

Happy to make changes if needed. Just wanna double check

Reply:
@gsouquet: Yes, @jpmedley is our human reference for Chromia like Opera :-)